src/documents/TextToSpeechAPI.md

```diff
@@ -37,9 +39,9 @@ Use the following function to create a Text To Speech component:
             <th>Parameters</th>
             <td>
                 <dl>
-                    <dt>options</dt>
+                    <dt>model.utteranceOpts</dt>
                     <dd>
-                        An optional data structure that configures the Text To Speech component, as described below.
+                        An optional data structure supplied to the component model that configures the Text To Speech component behaviour, as described below. As part of the component model, can be managed through the [ChangeApplier API](ChangeApplierAPI.md) to assist in coordination with other components.
```

Comment:
The link for the change applier added here needs to be specified using HTML not MD.